src/components/dev-hub/card-list.js

```diff
@@ -41,71 +40,79 @@ const getThumbnailUrl = media => {
         : media.thumbnailUrl;
 };
 
-// TODO: Update this component to take one array of items and map to cards based
-// on the item type
+const sortCardsByDate = contentList =>
+    contentList.sort((a, b) => {
+        return (
+            new Date(b.publishDate || b.pubdate) -
+            new Date(a.publishDate || a.pubdate)
+        );
+    });
+
+const showArticle = article => (
+    <ArticleCard
+        to={article['slug']}
+        key={article['_id']}
+        image={withPrefix(article['atf-image'])}
+        tags={getTagLinksFromMeta(article)}
+        title={getNestedText(article['title'])}
+        badge="article"
+        description={getNestedText(article['meta-description'])}
+    />
+);
+
+const showVideo = video => (
+    <VideoCard
+        key={video.title}
+        image={getThumbnailUrl(video)}
+        videoModalThumbnail={getThumbnailUrl(video)}
+        title={video.title}
+        badge={video.mediaType}
+        description={video.description}
+        video={video}
+    />
+);
+
+const showPodcast = podcast => (
+    <ArticleCard
+        key={podcast.title}
+        image={getThumbnailUrl(podcast)}
+        title={podcast.title}
+        badge={podcast.mediaType}
+        description={podcast.description}
+    />
+);
+
+const renderContentTypeCard = item => {
+    if (item.mediaType)
+        switch (item.mediaType) {
+            case 'youtube':
+            case 'twitch':
+                return showVideo(item);
```

Comment:
Sorry to nitpick on these but can we use `render` instead of `show`?
```suggestion
                return renderVideo(item);
```

Reply:
render definitely sounds better!